packages/code-connect/components/Breadcrumbs/ExpandableBreadcrumbs.figma.tsx

```diff
@@ -0,0 +1,16 @@
+import figma from '@figma/code-connect';
+import { BreadcrumbItem } from '@patternfly/react-core';
+
+figma.connect(
+  BreadcrumbItem,
+  'https://www.figma.com/design/aEBBvq0J3EPXxHvv6WgDx9/PatternFly-6--Components-Test?node-id=19922-43760',
+  {
+    props: {
+      children: figma.children('*')
+    },
+    example: (props) => (
+      // Documentation for BreadcrumbItem can be found at https://www.patternfly.org/components/breadcrumb
+      <BreadcrumbItem isDropdown>{props.children}</BreadcrumbItem>
```

Comment:
in figma, if they select that they want a dropdown, do they have to pick the dropdown? adding this prop doesn't create a dropdown, just changes some padding, so if we need to ship the dropdown as part of the code here, that'll need to be added.

Reply:
Good point, the menuToggles/dropdowns will be rendered as children in this case, we shouldn't need `isDropdown`

Reply:
if they are passing a dropdown as children, then they will need the `isDropdown` prop.